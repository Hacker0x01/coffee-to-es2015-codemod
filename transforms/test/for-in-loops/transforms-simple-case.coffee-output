var a, i, len;

for (i = 0, len = b.length; i < len; i++) {
  a = b[i];
  console.info(a);
}
