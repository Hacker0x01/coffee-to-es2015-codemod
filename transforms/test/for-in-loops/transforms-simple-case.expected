var a, i, len;

for (let a of b) {
  console.info(a);
}
